test_conformance/c11_atomics/host_atomics.h

```diff
@@ -112,8 +121,8 @@ CorrespondingType host_atomic_fetch_add(volatile AtomicType *a, CorrespondingTyp
     }
     else
     {
-#if defined( _MSC_VER ) || (defined( __INTEL_COMPILER ) && defined(WIN32))
-        return InterlockedExchangeAdd(a, c);
+#if defined(_MSC_VER) || (defined(__INTEL_COMPILER) && defined(WIN32))
+        return InterlockedExchangeAdd(reinterpret_cast<volatile LONG *>(a), c);
```

Comment:
Curious: if you change Line 111 above to `else if constexpr` do you still need this casting change?